app/src/main/java/com/google/droidjump/GameConstants.java

```diff
@@ -16,10 +16,10 @@
 
 package com.google.droidjump;
 public interface GameConstants {
+
     int DROID_COUNT_ON_FULL_DROID_PICTURE = 9;
     int DROID_JUMPING_CHARACTER_INDEX = 4;
     int DROID_FIRST_STEP_INDEX = 5;
     int DROID_SECOND_STEP_INDEX = 6;
+    int SLEEP_TIME = 20;
```

Comment:
Let's provide a comment above each constant about what it means and what does this value represent.